denops/skkeleton/function/input.ts

```diff
@@ -186,3 +187,28 @@ export async function deleteChar(context: Context) {
     context.kakutei("\b");
   }
 }
+
+export async function affix(context: Context, key: string) {
+  // prefix
+  if (
+    context.state.type === "input" &&
+    !context.state.directInput &&
+    context.state.henkanFeed.length > 0 &&
+    ["okurinasi", "okuriari"].includes(context.state.mode)
+  ) {
+    await acceptResult(context, [">", ""], "");
+    await henkanFirst(context, key);
+    return;
+  }
+
+  // suffix
+  if (context.state.type == "henkan") {
+    await kakutei(context);
+    henkanPoint(context);
+    await acceptResult(context, [">", ""], "");
+    return;
+  }
```

Comment:
It might be better to move this to `henkan.ts` with a name `suffix()`.
However, I did not do it at the moment because `acceptResult()` is not exported and I could not replace it with an exported `kanaInput()` function.

If we can split the function, I would also rename `affix()` to `prefix()`.

Reply:
I simply haven't exported it because I don't use it anywhere else. It's okay to change export it.